AFQ/api.py

```diff
@@ -210,6 +211,7 @@ def __init__(self,
                  scalars=["dti_fa", "dti_md"],
                  wm_labels=[250, 251, 252, 253, 254, 255, 41, 2, 16, 77],
                  use_prealign=True,
+                 use_xvfbwrapper=False,
```

Comment:
How about a more explicit name? Also, we should add this to the docstring.


```suggestion
                 virtual_frame_buffer=False,
```